salt/elife-bot/init.sls

```diff
@@ -63,6 +63,22 @@ elife-bot-crossref-cfg:
         - require:
             - elife-bot-repo
 
+elife-bot-pubmed-cfg:
+    file.managed:
+        - user: {{ pillar.elife.deploy_user.username }}
+        - name: /opt/elife-bot/pubmed.cfg
+        - source: salt://elife-bot/config/opt-elife-bot-pubmed.cfg
```

Comment:
this is the expected convention, it's simply the desired path with `/` replaced by `-`

Reply:
yes, it's just something I made up to keep the namespace shallow and it's contents obvious. it's not enforced by salt or anything